test/releases/99.0.0.go

```diff
@@ -79,7 +79,7 @@ func init() {
 				ParentName: "secondary-dns",
 				ParentKind: "Deployment",
 				Name:       "secondary-dns",
-				Image:      components.CoreDNSImageDefault,
+				Image:      "k8s.gcr.io/coredns/coredns@sha256:5b6ec0d6de9baaf3e92d0f66cd96a25b9edbce8716f5f15dcd1a616b3abd590e",
```

Comment:
main should have other hash IIRC
[registry.k8s.io/coredns/coredns@sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e](http://registry.k8s.io/coredns/coredns@sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e)

Reply:
ah nvm it is not main right?

Reply:
but i still dont understand

since KSD on 0.85.0 is 0.0.13 like main

https://github.com/kubevirt/cluster-network-addons-operator/blob/release-0.85/components.yaml#L13

it should be the same as main

Reply:
We will issue a new patch as soon as all the CVEs planned for this release are ready.. 
But I'm afraid I didn't catch your drift on the rest.
What image URL would you like me to use on 99.0.0?

Reply:
It fails on my local (soon to be seen on the lane) - it expects `k8s.gcr.io/coredns/coredns@sha256:5b6ec0d6de9baaf3e92d0f66cd96a25b9edbce8716f5f15dcd1a616b3abd590e` to be on the container.. 
Perhaps we should also change the manifest itself?

Reply:
It expects
registry.k8s.io/coredns/coredns@sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e

because the HEAD of 0.85 is v0.0.13
running fixed bump should fix you that

Reply:
I ran `make bump-kube-secondary-dns` but it did not fix it

Reply:
with the fixed bump ?
then something is wrong with the bump still, if the HEAD of 0.85 release has components.yaml with 0.0.13
this what should be focused on imo

Reply:
Problem here is that the prefix bump expect is registry.k8s.io/coredns/coredns
please replace the prefix only and run bump, then bump will fix it

pkg/components/components.go:   CoreDNSImageDefault               = "k8s.gcr.io/coredns/coredns@sha256:5b6ec0d6de9baaf3e92d0f66cd96a25b9edbce8716f5f15dcd1a616b3abd590e

Reply:
ACK. indeed fixed the thing.